docs/DeveloperGuide.md

```diff
@@ -152,15 +153,51 @@ Given below is an example usage scenario of how AddHousekeeperCommand behaves at
 
 Step 1: User launches the application for the first time. The list of housekeeper in records is empty.
 
-Step 2: User give an add housekeeper command `add housekeeper sally / 40`. The input will be split by the delimiter `/` to derive `sally` and `40`. The `CommandParser` runs parse which will return a new `AddHousekeeperCommand`.
+Step 2: User give an add housekeeper command `add housekeeper susan / 33`. The input will be split by the delimiter
+`/` to derive `susan` and `33`. The `CommandParser` runs parse which will return a new `AddHousekeeperCommand`.
 
-Step 3: `AddHousekeeperCommand#extractDetails()` will call `AddHousekeeperCommand#extractName()` and `AddHousekeeperCommand#extractAge()` to derive `sally` and `40`.
+Step 3: `AddHousekeeperCommand#extractDetails()` will call `AddHousekeeperCommand#extractName()` 
+and `AddHousekeeperCommand#extractAge()` to derive `susan` and `33`.
 
-Step 4: The `AddHousekeeperCommand` will now contain `sally` and `40`.
+Step 4: The `AddHousekeeperCommand` will now contain `susan` and `33`.
 
-Step 5: The execute method will derive the housekeeper name, `susan` which would call `HousekeeperList#hasNameAdded()` to verify if name has been recorded. If name is not in records, `HousekeeperList#addHousekeeper()` would be called to add the housekeeper, `sally` and `40` into the housekeeperList.
+Step 5: The execute method will derive the housekeeper name, `susan` which would call `HousekeeperList#hasNameAdded()` 
+to verify if name has been recorded. If name is not in records, `HousekeeperList#addHousekeeper()` would be called to 
+add the housekeeper, `susan` and `33` into the housekeeperList.
+
+![Sequence](team/falicia_addHousekeeperCommand/sequenceAddHousekeeperCommandv2.jpg)
+
+### Delete Housekeeper feature 
```

Comment:
Great explanation, but maybe could cut down on the level of technicality to make it more readable?